website/docs/r/storage_account.html.markdown

```diff
@@ -136,14 +136,16 @@ The following arguments are supported:
 
 * `queue_properties` - (Optional) A `queue_properties` block as defined below.
 
-~> **NOTE:** `queue_properties` cannot be set when the `account_kind` is set to `BlobStorage`
+~> **NOTE:** `queue_properties` can only be set for `Standard` (`account_tier`) `StorageV2` (`account_kind`) Storage Accounts.
 
 * `static_website` - (Optional) A `static_website` block as defined below.
 
 ~> **NOTE:** `static_website` can only be set when the `account_kind` is set to `StorageV2` or `BlockBlobStorage`.
 
 * `share_properties` - (Optional) A `share_properties` block as defined below.
 
+~> **NOTE:** `share_properties` can only be set for `Standard` (`account_tier`) `StorageV2` (`account_kind`) or `Premium` (`account_tier`) `FileStorage` (`account_kind`) Storage Accounts.
```

Comment:
(as above) I think we could probably reword this to be clearer, are these two combinations (Standard+StorageV2 and Premium+FileStorage)? If so:

```suggestion
~> **NOTE:** `share_properties` can only be configured when either `account_tier` is `Standard` and `account_kind` is `StorageV2` - or when `account_tier` is `Premium` and `account_kind` is `FileStorage`.
```